packages/myst-frontmatter/src/frontmatter/types.ts

```diff
@@ -24,6 +25,44 @@ export type Biblio = {
   last_page?: string | number;
 };
 
+export type Thebe = {
+  useBinder?: boolean;
```

Comment:
Can you use both `binder` and `jupyter-lite`, i.e. can both of these be `true`?

Reply:
Ah, I see, all of these come directly from the `thebe` options, so they probably should continue to match that.

Reply:
Yes both can be true, and from `thebe-react` binder would win. probably something that could be made better but the `useBinder` is a legacy thebe option.

Reply:
@fwkoch redefining the types here is right? i mean rather than adding `thebe-core` as a type only dependency?

Reply:
`myst-frontmatter` is quite lightweight and imported in many places, so we do not want to import `thebe-core` here.

You could go the other way - make this the source of truth for the types and add `myst-frontmatter` as a dependency to `thebe-core`? Or just redefine them - doing that gives some flexibility to be different (maybe useful if, say, the theme is setting a thebe option that we do not want frontmatter to control, or something...).